apps/frontend/src/pages/auth/auth.tsx

```diff
@@ -43,8 +46,23 @@ const Auth: React.FC = () => {
 
 	return (
 		<>
-			state: {dataStatus}
-			{getScreen(pathname)}
+			{/* state: {dataStatus} */}
+			<main className="auth-container">
```

Comment:
Maybe I'm wrong, but it seemed to me that classes should be written like this clasName={styles["class"]}??

Reply:
actually this new knowledge for me, thank you for suggestion